orbit/pkg/bitlocker/bitlocker_management.go

```diff
@@ -1,17 +1,94 @@
 package bitlocker
 
-// Encryption Status
+// Volume encryption/decryption status.
+//
+// Values and their meanings were taken from:
+// https://learn.microsoft.com/en-us/windows/win32/secprov/getconversionstatus-win32-encryptablevolume
+const (
+	CONVERSION_STATUS_FULLY_DECRYPTED        int32 = 0
+	CONVERSION_STATUS_FULLY_ENCRYPTED        int32 = 1
+	CONVERSION_STATUS_ENCRYPTION_IN_PROGRESS int32 = 2
+	CONVERSION_STATUS_DECRYPTION_IN_PROGRESS int32 = 3
+	CONVERSION_STATUS_ENCRYPTION_PAUSED      int32 = 4
+	CONVERSION_STATUS_DECRYPTION_PAUSED      int32 = 5
+)
+
+// Free space wiping status.
+//
+// Values and their meanings were taken from:
+// https://learn.microsoft.com/en-us/windows/win32/secprov/getconversionstatus-win32-encryptablevolume
+const (
+	WIPING_STATUS_FREE_SPACE_NOT_WIPED          int32 = 0
+	WIPING_STATUS_FREE_SPACE_WIPED              int32 = 1
+	WIPING_STATUS_FREE_SPACE_WIPING_IN_PROGRESS int32 = 1
+	WIPING_STATUS_FREE_SPACE_WIPING_PAUSED      int32 = 1
```

Comment:
I believe `InProgress` should be `2` and `Paused` should be `3`?

Reply:
awesome catch, thank you!!